src/datagrid/DataGrid.tsx

```diff
@@ -1071,7 +1071,7 @@ export class DataGrid extends React.PureComponent<DataGridProps, DataGridState>
     private buildDataGridBody(): React.ReactElement {
         const {allRows} = this.state;
         return (
-            <div className={ClassNames.DATAGRID} style={{overflow: "initial"}}>
+            <div className={ClassNames.DATAGRID}>
```

Comment:
Datagrid Footer is broken due to some changes. Please don't add custom inline styles to fix it we need to fix original issue.

Reply:
@Bhatii along with this please also remove inline style for Daragrid header that u have added in PR https://github.com/EMCECS/clarity-react/pull/244